### ISSUE TITLE
Arch Linux package builds, but errors when run

### ISSUE THREAD

**Issue by fryfrog** [andreasnuesslein/runrestic#23]
If I'm lucky, you'll recognize exactly what I'm doing wrong. Otherwise, I'll continue to dig into what is going on... probably something silly on my part. ;)

```
0 ✓ fryfrog@apollo ~ $ runrestic
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2451, in resolve
    return functools.reduce(getattr, self.attrs, module)
AttributeError: module 'runrestic.runrestic.runrestic' has no attribute 'main'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/runrestic", line 11, in <module>
    load_entry_point('runrestic==0.5.0', 'console_scripts', 'runrestic')()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2453, in resolve
    raise ImportError(str(exc))
ImportError: module 'runrestic.runrestic.runrestic' has no attribute 'main'
```

**Comment by andreasnuesslein** [comment]
https://github.com/andreasnuesslein/runrestic/tree/setuppy_fix

I think this was the problem - would you mind checking it? Cheers

**Comment by fryfrog** [comment]
Yup, that fixes it. I just modified `/usr/lib/python3.8/site-packages/runrestic-0.5.0-py3.8.egg-info/entry_points.txt` `main` -> `runrestic` and it runs fine. :)

```
[console_scripts]
runrestic = runrestic.runrestic.runrestic:runrestic
```

Thanks for realizing what it was, because I had nothing. :)

**Comment by fryfrog** [comment]
Confirmed fixed in `0.5.2`, thanks. :)
